dev/domain-controller/new-admin-vm/Install-CAM.ps1

```diff
@@ -1329,15 +1331,13 @@ brokerLocale=en_US
 				& "keytool" -import -file "$env:systemdrive\$issuerCertFileName" -keystore ($env:classpath + "\security\cacerts") -storepass changeit -noprompt
                 $ErrorActionPreference = $eap
 
-		        Write-Host "Finished! Restarting Tomcat service for $using:brokerServiceName."
-
-				Restart-Service $using:brokerServiceName
```

Comment:
Not starting the services (ever) since the final step will reboot the machine and start the services that way. They're both set to Automatic.